### DISCUSSION TITLE
Scrollbar Styling

### DISCUSSION THREAD

**Discussion by moishinetzer** [tailwindlabs/tailwindcss#8269] [Ideas]
With the standard scrollbar proposal moving along, and with webkit options as shown in 

https://css-tricks.com/almanac/properties/s/scrollbar-color/

Are we at a stage where we can implement this into the standard tailwind package as prefixes such as: scroll-bg-green-200
scroll-text-blue-200 for the track and thumb respectively? Or perhaps even thumb-x track-x

**Comment by ghost** [comment]
this feature should be added in tailwind for developer use

**Reply by cnsource** [reply]
yes, much needed

**Reply by SakarDev** [reply]
Yes please we need it.!

**Comment by lukewarlow** [comment]
In case anyone comes across this discussion looking for a solution I've made an npm package with utility plugins for scrollbar-color, scrollbar-width and scrollbar-gutter. See https://github.com/lukewarlow/tailwind-scrollbar-utilities

This only uses the CSS standard properties and doesn't allow for styling using the deprecated ::-webkit-scrollbar CSS as that isn't supported in Firefox and has a much more complicated API.

I do however use this legacy CSS within scrollbar-none to improve browser support.
